src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,51 @@
 package core.basesyntax;
 
-class Pair {
+public class Pair<T,S> {
+    private T first;
+    private S second;
 
+    private Pair(T first,S second) {
+        if (first != null && second != null) {
+            this.first = first;
+            this.second = second;
+        }
```

Comment:
if I want to have Pair object where first = "some value" but second = null. Why can't I?

Reply:
made mistake due wrong understanding task condition "and does not prevent elements from being null". Done.